src/components/sidebar/routePathView/LinkListView.tsx

```diff
@@ -0,0 +1,195 @@
+import * as React from 'react';
+import { inject, observer } from 'mobx-react';
+import { SidebarStore } from '~/stores/sidebarStore';
+import { IRoutePath, INode, IRoutePathLink } from '~/models';
+import routeBuilder  from '~/routing/routeBuilder';
+import subSites from '~/routing/subSites';
+// import navigator from '~/routing/navigator';
+import NodeType from '~/enums/nodeType';
+import ButtonType from '~/enums/buttonType';
+import { Dropdown, Button } from '../../controls';
+import * as s from './linkListView.scss';
+// import classnames from 'classnames';
+
+interface ILinkListViewProps {
+    sidebarStore?: SidebarStore;
+    routePath: IRoutePath;
+}
+
+interface ILinkListViewState {
+    selectedRoutePathLink?: string;
+    linkTableFilter: string;
+}
+
+const filterTypes = {
+    STOPS: 'Pysäkki',
+    DISABLED: 'Pysäkki, joka ei ole käytössä',
+    CROSSROAD: 'Risteys',
+    BORDER: 'Raja',
+    ALL: 'Näytä kaikki',
+};
+
+@inject('sidebarStore')
+@observer
+class ILinkListView extends React.Component<ILinkListViewProps, ILinkListViewState>{
+    constructor(props: any) {
+        super(props);
+        this.state = {
+            selectedRoutePathLink: undefined,
```

Comment:
This row is I think unnecessary, selectedRoutePathLink is by default undefined even without this row

Reply:
removed